app/views/users/confirm_credentials.html.erb

```diff
@@ -27,6 +27,9 @@
               <%= "Join accounts and login" %>
           <% end %>
       <% end %>
+      <ul>
+        <li><%= link_to "Forgot your password?", new_user_password_path %></li>
```

Comment:
use the translation provided in yml files. You can see that in other places where this button is placed, like the homepage